src/containers/BooksList.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import Book from '../components/Book';
+
+const BooksList = ({ books }) => (
+  <table>
+    <thead>
+      <tr>
+        <td>Book ID</td>
+        <td>Title</td>
+        <td>Category</td>
+      </tr>
+    </thead>
+    <tbody>
+      { books.map(book => (
```

Comment:
**OPTIONAL**: You can desctructure on the argument to clean this a bit, not a huge deal anyhow.

```js
{books.map(({ id, title, category }) => (
  <Book key={id} id={id} title={title} category={category} />
))}
```

Reply:
Nice! Thank you very much :D